inc/config.php

```diff
@@ -47,33 +47,19 @@
 
 Config::detectRootDoc();
 
+loadGlpiConfig((isset($USEDBREPLICATE) ? $USEDBREPLICATE : 0),
+      (isset($DBCONNECTION_REQUIRED) ? $DBCONNECTION_REQUIRED : 0),
+      (isset($TRY_OLD_CONFIG_FIRST) ? 1 : 0));
```

Comment:
in the original code $TRY_OLD_CONFIG_FIRST is a global var and is tested with isset() only.

Introducting functions, i found safer to use a value 0 or 1 as the other arguments rather than just relying on isset(). This way the 3 args of the function are homogeneous.